app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt

```diff
@@ -26,17 +28,18 @@ import org.fossasia.susi.ai.R
 import org.fossasia.susi.ai.chat.ChatActivity
 import org.fossasia.susi.ai.dataclasses.FetchFeedbackQuery
 import org.fossasia.susi.ai.dataclasses.PostFeedback
+import org.fossasia.susi.ai.dataclasses.ReportSkillQuery
 import org.fossasia.susi.ai.helper.PrefManager
 import org.fossasia.susi.ai.helper.Utils
+import org.fossasia.susi.ai.skills.SkillsActivity
 import org.fossasia.susi.ai.rest.responses.susi.GetSkillFeedbackResponse
 import org.fossasia.susi.ai.rest.responses.susi.SkillData
 import org.fossasia.susi.ai.rest.responses.susi.Stars
-import org.fossasia.susi.ai.skills.SkillsActivity
```

Comment:
@codedsun It's been around a month since this feature is being worked upon. I have pointed out several times in my reviews not to make such unrelated changes in PRs. Still, I find such changes here. :( 
Please, remove all these changes. And, also review your own PR before requesting a final review.

Reply:
@arundhati24 These auto optimisation feature for android studio, does this all the time to remove the classes, which are not being used. Will be updating it again.

Reply:
Still the same line is present on Line 34